lib/Authy/AuthyApi.php

```diff
@@ -41,12 +41,9 @@ class AuthyApi
     public function __construct($api_key, $api_url = "https://api.authy.com")
     {
         $this->rest = new \GuzzleHttp\Client(array(
-            'base_url' => "{$api_url}/protected/json/",
-            'defaults' => array(
-                'headers'       => array('User-Agent' => $this->__getUserAgent() ),
-                'query'         => array('api_key' => $api_key),
```

Comment:
Yes, it was tested. I don't see anything in Client.php in the Guzzle repo that actually says it uses the `defaults` subarray. The phpdoc merely says that any request option can be passed as a default. `defaults` isn't a request option: `headers` is.

I think the unit tests would have failed if this wasn't the correct way to set the defaults, by the way.